node/network/src/peer_manager.rs

```diff
@@ -129,14 +136,39 @@ impl PeerManager {
                     all_peer_states3.clone(),
                     inc_msg_tx.clone(),
                     reconnect_delay,
+                    chain_state_retriever.clone(),
```

Comment:
This will send chain info only from one peer to another. Oppose to having it sent both ways, like in a real exchange.